[2025-03-13T12:50:00.813+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data_etl.extract_weather_task scheduled__2025-03-13T12:45:00+00:00 [queued]>
[2025-03-13T12:50:00.820+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data_etl.extract_weather_task scheduled__2025-03-13T12:45:00+00:00 [queued]>
[2025-03-13T12:50:00.820+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-03-13T12:50:00.829+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): extract_weather_task> on 2025-03-13 12:45:00+00:00
[2025-03-13T12:50:00.835+0000] {standard_task_runner.py:60} INFO - Started process 567 to run task
[2025-03-13T12:50:00.837+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_data_etl', 'extract_weather_task', 'scheduled__2025-03-13T12:45:00+00:00', '--job-id', '45', '--raw', '--subdir', 'DAGS_FOLDER/etl_repeat_weather.py', '--cfg-path', '/tmp/tmpbh03k1qa']
[2025-03-13T12:50:00.839+0000] {standard_task_runner.py:88} INFO - Job 45: Subtask extract_weather_task
[2025-03-13T12:50:00.878+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_data_etl.extract_weather_task scheduled__2025-03-13T12:45:00+00:00 [running]> on host 7de7158dc909
[2025-03-13T12:50:01.110+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data_etl' AIRFLOW_CTX_TASK_ID='extract_weather_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-13T12:45:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-13T12:45:00+00:00'
[2025-03-13T12:50:01.113+0000] {logging_mixin.py:188} INFO - Menggunakan data dummy karena API key tidak diatur.
[2025-03-13T12:50:01.134+0000] {python.py:201} INFO - Done. Returned value was: Data cuaca berhasil diambil pada 2025-03-13 12:50:01 untuk 5 kota
[2025-03-13T12:50:01.150+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_data_etl, task_id=extract_weather_task, execution_date=20250313T124500, start_date=20250313T125000, end_date=20250313T125001
[2025-03-13T12:50:01.170+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-03-13T12:50:01.191+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
